### PR TITLE
Feat/hng 14 create preview page

### PR DESCRIPTION
**Pull request by favouridoko1**
# Description

**Closes #issue_number_here**

# Changes proposed

## What did you do?

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos

### CODE REVIEW

**Comment by phoenixdahdev** [comment]
Hi @favouridoko1 I don't have access on the remix repo

**Comment by favouridoko1** [comment]
Alright @incredible-phoenix246  thanks for responding

File: package-lock.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
         "clsx": "^2.1.1",
         "isbot": "^4.4.0",
         "lucide-react": "^0.408.0",
+        "pnpm": "^9.6.0",
```
Comment (abiona01) [inline package-lock.json]: uninstall pnpm

**Review by abiona01 — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (favouridoko1) [reply package-lock.json]: I have uninstalled in with the commad i "pnpm remove pnpm" command 
 please kindely checkback

Comment (favouridoko1) [reply package-lock.json]: Screenshot 2024-07-23 092749

**Review by abiona01 — APPROVED** [review]

**Pull request merged**